### PR TITLE
Feat/camera

### PR DESCRIPTION
**Pull request by ThinhKimVo**

### CODE REVIEW

**Review by loido — APPROVED** [review]

File: packages/webrtc-firebase/src/webrtc_firebase_proxy.ts [hunk 1]
```diff
@@ -135,29 +183,36 @@ class WebRTCFirbase extends Base implements IVideoCall {
...
+  };
 
   setup = async () => {
     console.log("Setup webrtc");
-
     try {
-      this.peerConnection = new RTCPeerConnection(peerConstraints);
+      const peerConnection = new RTCPeerConnection(peerConstraints);
+
+      this.peerConnection = peerConnection
```
Comment (loc-nguyenthien) [inline packages/webrtc-firebase/src/webrtc_firebase_proxy.ts:195]: This change seems not necessary

File: src/provider/proxy/constants.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const COLLECTION_PATHS = {
```
Comment (loc-nguyenthien) [inline src/provider/proxy/constants.ts:1]: dupplicate values with file [packages/webrtc-firebase/src/constants.ts](https://github.com/saigontechnology/rn-video-call/pull/6/files#diff-9d36b1c76d2cd1798a531b5692e99b2c9eafd5e74ba4fcb4c510d288ff294558)

File: tsconfig.json [hunk 3]
```diff
@@ -4,6 +4,6 @@
   "compilerOptions": {
     "outDir": "./build"
   },
-  "include": ["./src"],
+  "include": ["./src", "example/components"],
```
Comment (loc-nguyenthien) [inline tsconfig.json:7]: I think we should move `components` to a separate place, if we need to use them later in our real project, and not put them into `example`

Comment (ThinhKimVo) [reply tsconfig.json:7]: just a temporary, we will remove when packages repo is ready.

File: src/provider/proxy/constants.ts (hunk 2 above)
Comment (ThinhKimVo) [reply src/provider/proxy/constants.ts:1]: just a temporary, we will remove when packages repo is ready.

File: packages/webrtc-firebase/src/webrtc_firebase_proxy.ts (hunk 1 above)
Comment (ThinhKimVo) [reply packages/webrtc-firebase/src/webrtc_firebase_proxy.ts:195]: just a temporary, we will remove when packages repo is ready.

**Pull request merged**